pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -374,8 +380,11 @@ def _setup_execution_config(
 
         updated_values = {}
 
+        _device_options = (
+            self._device_options_mps if self.method == "mps" else self._device_options_tn
+        )
```

Comment:
Excellent candidate for:
```suggestion
        _device_options = _device_options[self.method] 
```
if using a dictionary.

Reply:
Done!